mods/ra/maps/allies-05a/weapons.yaml

```diff
@@ -2,19 +2,14 @@ MissionColt:
 	ReloadDelay: 6
 	Range: 7c0
 	Report: gun5.aud
-	Projectile: Bullet
-		High: true
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
 		Spread: 42
 		Damage: 50
 		Versus:
-			None: 0
 			Wood: 10
-			Light: 0
-			Heavy: 0
-			Concrete: 0
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
+		AffectsParent: true
 	Warhead@2Eff: CreateEffect
 		Explosions: piffs
 		InvalidImpactTypes: Water
```

Comment:
In the original there were multiple shots in rapid succession with a bit of (visual only?) inaccuracy, but the weapon itself definitely displayed only one piff per shot.

Anyway, I can live with keeping the damage since it's so minor, but the shooting is happening inside the building so there shouldn't be any visible piffs.